src/components/layout/CommentInput.jsx

```diff
@@ -8,28 +8,37 @@ const CommentInput = () => {
   const [message, setMessage] = useState('');
   const [focus, setFocus] = useState(false);
 
-  const submitComment = (event) => {
-    event.preventDefault();
+  const submitComment = () => {
     if (subscriber) {
       const comments = {
-        body: event.target.body.value,
+        body: input,
       };
-      Articles.createComment(comments, article.id);
-      setInput('');
-      setMessage('');
-      setFocus(false);
+      if (input.trim().length !== 0) {
+        Articles.createComment(comments, article.id);
+        setInput('');
```

Comment:
Why the trim? :P

Reply:
to get rig of the white spaces. so u can't write like 2 spaces in a row and not get the message me set for empty input :3